pkg/apis/pipeline/v1beta1/task_validation_test.go

```diff
@@ -33,13 +33,6 @@ import (
 	"knative.dev/pkg/apis"
 )
 
-var validResource = v1beta1.TaskResource{
-	ResourceDeclaration: v1beta1.ResourceDeclaration{
-		Name: "validsource",
-		Type: "git",
```

Comment:
can you change this to an image resource instead of removing these tests?

Reply:
Thanks for catching!

Sorry that some of the restoring of git type test cases are removed during multi-state rebasing.